index.md

```diff
@@ -1,51 +1,70 @@
-# Health Record Ingestion service
+# Introduction
 
-The HRI is a deployment ready service for streaming Health-related data into the cloud. It provides a "front door" for "Data Integrators" to send data into the IBM Cloud, while supporting both batch-processing and data streaming workflows. It provides features to initiate and track the movement of a dataset into the IBM cloud for both ["Data Integrators"](glossary.md#data-integrator) and ["Data Consumers"](glossary.md#data-consumer). 
+The **[Health Record Ingestion service](glossary.md#health-record-ingestion-hri)** is a deployment-ready service for streaming healthcare-related data into the cloud. It provides a "front door" for [Data Integrators](glossary.md#data-integrator) to send data into the IBM&reg; Cloud, while supporting both batch processing and data streaming workflows. It provides features to initiate and track the movement of a dataset into the IBM Cloud for both Data Integrators and [Data Consumers](glossary.md#data-consumer). 
```

Comment:
Link still doesn't work. I might have had it incorrect last time.  Should be `health-record-ingestion-service-hri`.